bin/shpec

```diff
@@ -22,7 +22,7 @@ describe() {
 end() {
   : $((test_indent -= 1))
   if [ $test_indent -eq 0 ]; then
-    [ $failures -eq 0 ]
+    [ "$failures" -eq 0 ]
```

Comment:
@rylnd - yer no idea - and the same line occurs in `final_results` - so i don't have 100% confidence in this linter, but at least it has *some* useful suggestions.